example_buffer.py

```diff
@@ -25,12 +25,12 @@
 
 LOCAL_DIR = "data/"
 
-MINIMUM_NEW_GAMES = 5000
+MINIMUM_NEW_GAMES = 12000
 
 
 def pick_examples_from_tfrecord(filename, samples_per_game=4):
     protos = list(tf.python_io.tf_record_iterator(filename, READ_OPTS))
-    if len(protos) < 20:  # Filter games with less than 20 moves
+    if len(protos) < 50:  # Filter games with less than 20 moves
         return []
     choices = random.sample(protos, min(len(protos), samples_per_game))
```

Comment:
not going to go with the len(protos) // 50 idea?

Reply:
`// 50` is for when a game is in the window for ~50 generations, correct?  That's probably not the ratio anymore...